recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "gensim" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "eb099de1e50447c42e168a1a99de4721923688afc71b12fe522f79687a4fbb13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.3
```

Comment:
Since gensim uses cython, i would change this to 

```
numpy x.x
cython
``` 
and change the numpy section in the run area too.

This will build you a version of gensim that is compatible with the numpy ABI for the versions that conda forge supports